test/DebugInfo/Inputs/local_type_originally_defined_in_other.swift

```diff
@@ -1,2 +1,8 @@
 @available(macOS 10, *)
```

Comment:
Would there also be a benefit in having a test that uses an originallyDefinedIn type that was imported from another module?
```
A.swift
_originallyDefinedIn(module: "B", macOS 10.1) 
public struct foo {}
C.swift
import A
let f = foo()
```